src/System.Runtime.Extensions/ref/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
-    "System.Runtime": "4.4.0-beta-24629-01"
+    "System.Runtime": "4.4.0-beta-24629-01",
+    "System.Security.Principal": "4.4.0-beta-24629-01"
```

Comment:
I'm not sure I like this dependency. If we want this we should consider folding System.Security.Principal into S.Runtime.Extensions instead.